sections/conclusion.tex

```diff
@@ -1 +1,29 @@
-\section{Conclusion}\label{sec:conclusion}
\ No newline at end of file
+\section{Conclusion}\label{sec:conclusion}
+We explore possibilities of incorporating meta-data into existing topic models, such as \gls{lda} and \gls{pam}.
+We evaluate three different metadata, Author, Category, and Taxonomy, each of which represent the data in a different way.
+From the topic coherence results, shown in \autoref{tab:metric_results}, the \gls{pam} model using the Taxonomy meta data gets the best results.
+The Author and Category models are the worst performing, where the topic coherence is much lower than the other models.
+We want to answer the problem statement, which we stated in \autoref{sec:introduction}:
+
+\begin{itemize}
+	\item \textit{How does including metadata within the \gls{lda} model impact the resulting topics?}
+	\item \textit{What possible problems can these models help alleviate at Nordjyske?}
+	\begin{itemize}
+		\item \textit{More specifically, how can these models improve recommendation?}
+	\end{itemize}
+\end{itemize}
+
+We incorporate metadata into our topic models in multiple ways.
+The \gls{lda} model is used as a baseline, which indicates whether incorporating metadata can improve the topic quality of our topic models.
+Based on our analysis, we see that only using the metadata for topic assignment within \gls{lda}, can hurt the topic quality, which is how Author-Topic and Category-Topic is run.
+Other studies have shown, that only using the metadata for topic assignment, improves the performance.
+This might be due to the particularities of the dataset, such as authors usually do not write about the same subject within the news environment.  

```

Comment:
```suggestion
We incorporate metadata into our topic models in multiple ways.
The \gls{lda} model is used as a baseline, which indicates whether incorporating metadata can improve the topic quality of our topic models.
Based on our analysis, we see that only using the metadata for topic assignment within \gls{lda}, can hurt the topic quality, which is how Author-Topic and Category-Topic are run.
Other studies have shown, that only using the metadata for topic assignment, improves the performance.
This might be due to the particularities of the dataset, such as authors usually do not write about the same subject within the news environment.  

```